ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml

```diff
@@ -0,0 +1,19 @@
+build_root:
+  project_image:
+    dockerfile_path: Dockerfile.ci
+resources:
+  "*":
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+  - as: lint
+    commands: yarn run lint
+    container:
+      from: bin
```

Comment:
You need `binary_build_commands` to build the `bin` image (see https://docs.ci.openshift.org/docs/architecture/ci-operator/#building-artifacts)